specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+in ux, when considering batch size request and gas limitations, that might trip some developers up. 
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+The rationale for this design, is that we want to support developers that need batch requests. In many ecosystems, a single NFT or 
+series of NFT are not sufficient for representing and managing potentially 100s of tokens. Managing cross contract calls and the token contracts themselves become a huge burden on the developer. This will reduce the complexity, and allow developers to easily bridge over to other contracts like ERC-1155 on other chains that allow for a this style of representation.
+
+In the design phase it was considered to simply rely on the underlying implementations of fungible tokens and non-fungible tokens
+to be the scope of interaction with the chain. Doing this we would have tied the implementations to the bounds of FT and NFT. By loosing this up a bit, we are able to be a bit more flexible in what's possible.
+
+Not doing this means we really won't have a great way of supporting use cases where developers need to represent and manage
+large varying quantities and types of tokens. They will all have to implement it on their own, and that would make for a fragmented
+and weak ecosystem. Where every developer would not be able to reliably trade these assets.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+The unresolved questions, are really what type of metadata , is required for this spec?
+- We decided that having metadata on chain is good, and that at later time we can have a spec for extra data if needed
+
+Can we represent events in this spec, which would improve the ecosystem quite a bit? If we represent events what should those events be?
+- We decided events aren't here yet in the ecosystem, in the way we' like them to be so they are't apart of this standard
+
+Should we have a spec for TokenType?
+- We decided TokenType shouldn't be exposed for the public interface consumption. We will leave this description in the hands of the implementers
+
+Should we have a spec for offchain metadata?
+- We decided no spec for offchain metadata yet but maybe in the future
+
+Does the current storage management scheme work for people?
+- The current storage management scheme works for folks
+
+How freeform should this token be? Right now there is a notion of supply, which is not 100% guaranteed every token has
+or wants to track supply semantics. Not having supply makes everything more difficult and requires consumers of the contract
+to track minting and burning events.
+- This point was taken to show that due to lack of proper events, and having the additional capability of storing this on chain, would
+result in a reduction of complexity for consumers of the contract data.
+
+Approval Management is probably out of the scope of this solution.
```

Comment:
Is it out of scope or not?  There are references to approval ids and approvers in this document -- if it's out of scope, we should not have any references to it.  I think it's also worth carefully considering this decision -- if **not** having approval management in-scope makes it impossible for current NEP-141 or NEP-171 contracts to migrate to this standard if they use approval management, then it really isn't doing what it set out to do IMHO and will lead to splintering of contract implementations due to a lack of support for this functionality (which will result in people rolling their own NEP141 and NEP171 batching so they can perform batch transfers and keep their existing approval support; something that this standard specifically aims to avoid)

Reply:
Nice catch! 

On approval management, we discussed this for a while too, and market participants within the discussions didn't see the need for approval management to the spec at this point in time. We decided that it could come later.  

I think the thought is that  this is to be an alternative tool in the wheelhouse. I think an example of this is the way  opensea operates allowing users to mint both erc-721 series, erc-721 one shots, and erc-1155 tokens, not quite a zero sum game. Additionally videogames have this same sort of exchange use case where it's useful to have a single contract mint and exchange vs many contracts per items. 

I think contract migration never came up, I think in part because we just viewed this a different tool. I'd be curious to know about any stats for approval management in contracts in the wild.  I started poking around at the indexer logs to just get a feel for who is using approval management in the wild. 

Let's bring this up at last call meeting for this, which is looking more like Wednesday.